clients/fides-consent/rollup.config.js

```diff
@@ -65,12 +65,6 @@ export default [
       esbuild({
         minify: !isDev,
       }),
-      copy({
-        // Automatically add the built script to the privacy center's static files for testing:
```

Comment:
note to self- pretty sure we still need this. This was previously added to `privacy-center/public`